include/tune_quda.h

```diff
@@ -15,8 +15,8 @@
 
 // this file has some workarounds to allow compilation using nvrtc of kernels that include this file
 #ifdef __CUDACC_RTC__
-#define CUresult bool
-#define CUDA_SUCCESS true
+#define QUresult bool
+#define QUDA_SUCCESS true
```

Comment:
These edit should not be made.  These defines are happening specifically when the CUDA run-time compiler is detected, and should not subject to any abstractions.

Reply:
Hmmm, on second thought, let me hold judgement on this.  Has a jitify build been verified in this branch yet>

Reply:
My auto converter got that. Fixed.